source/System.cpp

```diff
@@ -303,6 +308,11 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 				object.message = &UNINHABITEDPLANET;
 		}
 	}
+	
+    // Make sure that the stellarWindStrength and luminosity are not game brakeingly low or high.
+    stellarWindStrength = std::max(0.25, stellarWindStrength);
+    luminosity = std::max(0.5, luminosity);
+    luminosity = std::min(2.0, luminosity);
```

Comment:
I would clamp the upper range to 4 or 5 - several stars already have luminosity 2.
You should allow for 0 luminosity and 0 wind (a system without a star, nebulae, or perhaps one in which something is already capturing the entire output of the star(s)). A safety catch for purely solar-powered ships could be `luminosity = std::max(.001, luminosity);`, as this would allow them a tiny bit of generation (from the galactic background light).
I would not force positive values of stellarWindStrength, as a system that drains hyperspace fuel would be a neat tool for modders (a black hole system, or some alien artifact). I'd suggest enforcing a sane number though, e.g. `stellarWindStrength = std::max(-.05, stellarWindStrength);` so that it is not insanely draining.